Airdrop.test.js

```diff
@@ -0,0 +1,136 @@
+const {
+    BN,
+    send,
+    constants,
+    expectRevert,
+    assertRevert,
+    balance,
+    ether
+} = require('@openzeppelin/test-helpers');
+require('chai')
+    .use(require('chai-as-promised'))
+    .use(require('chai-bn')(BN))
+    .should();
+
+const Token = artifacts.require("Token");
+const Airdrop = artifacts.require("Airdrop");
+
+contract("Airdrop", ([owner, user1, user2, user3]) => {
+
+    beforeEach(async () => {
+        tokenInstance = await Token.new({ from: owner });
+        airdropInstance = await Airdrop.new(tokenInstance.address, { from: owner });
+    });
+
+    it("Should deposit ether to the contract", async () => {
+
+        await airdropInstance.depositEther({ from: owner, value: ether('2') });
+
+        let etherBalanceAirdrop = await balance.current(airdropInstance.address);
+
+        etherBalanceAirdrop.should.be.bignumber.equal(ether('2'));
+    });
+

```

Comment:
add next test for depositEther : "Should be fail if caller not owner"

Reply:
The procedure for checking the change in balance is as follows:
let beforeBalance = await balance.cu....
await operation....
let newBalance = await balance...
expect(beforeBalance).to.be.bignumber.equal(beforeBalance.add( "the amount that was transferred"));

Reply:
in order to avoid the situation that by this time the balance was already 2 ether